plotly/tests/test_core/test_plotly/test_plot.py

```diff
@@ -9,70 +9,63 @@
 
 import requests
 import six
+import json
 
 from unittest import TestCase
 from nose.plugins.attrib import attr
 from nose.tools import raises
 
+from plotly.tests.utils import PlotlyTestCase
```

Comment:
:cow2: Similar to above, we typically order `from` imports below `import` imports.

https://github.com/plotly/dev-docs/blob/master/reference/styleguides/python.md#imports